Cargo.toml

```diff
@@ -34,10 +34,13 @@ heck = "0.5.0"
 
 [dependencies]
 edgeql-parser = {git = "https://github.com/edgedb/edgeql"}
-gel-protocol = {git = "https://github.com/edgedb/edgedb-rust/", features=["all-types"]}
-gel-derive = {git = "https://github.com/edgedb/edgedb-rust/"}
-gel-errors = {git = "https://github.com/edgedb/edgedb-rust/"}
-gel-tokio = {git = "https://github.com/edgedb/edgedb-rust/", features=["admin_socket", "unstable"]}
+
+# Use cargo update for these
+gel-protocol = { version = "0", features = ["all-types"] }
+gel-derive = { version = "0" }
+gel-errors = { version = "0" }
+gel-tokio = { version = "0", features=["admin_socket", "unstable"] }
```

Comment:
A nitpick: this will make `cargo update` something that can break our code. I'd prefer if we included minor version here `0.7`.